docs/tables/aws_cost_by_tags.md

```diff
@@ -0,0 +1,66 @@
+# Table: aws_cost_by_tags
+
+Amazon Cost Explorer helps you visualize, understand, and manage your AWS costs and usage. The `aws_cost_by_tags` table provides a simplified view of cost by tags in your account. ou must specify a granularity (`MONTHLY`, `DAILY`) and `tag_key_1`. `tag_key_2` is optional for the table.
+
+Note that [pricing for the Cost Explorer API](https://aws.amazon.com/aws-cost-management/pricing/) is per API request - Each request will incur a cost of $0.01.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  tag_key_1,
+  tag_value_1,
+  period_start,
+  blended_cost_amount::numeric::money,
+  unblended_cost_amount::numeric::money,
+  amortized_cost_amount::numeric::money,
+  net_unblended_cost_amount::numeric::money,
+  net_amortized_cost_amount::numeric::money
+from
+  aws_cost_by_tags
+where
+  granularity = 'DAILY'
+and
+  tag_key_1 = 'Name';
+```
+
+### Min, Max, and average daily unblended_cost_amount by tag
+
+```sql
+select
+  tag_key_1,
+  tag_value_1,
+  min(unblended_cost_amount)::numeric::money as min,
+  max(unblended_cost_amount)::numeric::money as max,
+  avg(unblended_cost_amount)::numeric::money as average
+from
+  aws_cost_by_tags
+where
+  granularity = 'DAILY'
+and
+  tag_key_1 = 'Name'
+group by
+  tag_key_1, tag_value_1;
+```
+
+### Ranked - Top 10 Most expensive days (unblended_cost_amount) by tag
+
+```sql
```

Comment:
Im getting EOF errors (`Error: rpc error: code = Unavailable desc = error reading from server: EOF (SQLSTATE HV000)` running this with an aggregator...

Reply:
@johnsmyth I am not getting any error for a aggregator connection.

```
> with ranked_costs as (
  select
    tag_key_1,
    tag_value_1,
    period_start,
    unblended_cost_amount::numeric::money,
    rank() over(partition by tag_key_1 order by unblended_cost_amount desc)
  from
  aws_all.aws_cost_by_tag
where
  granularity = 'DAILY'
and
  tag_key_1 = 'Name'
)
select * from ranked_costs where rank <= 10;
+-----------+-------------+---------------------------+-----------------------+------+
| tag_key_1 | tag_value_1 | period_start              | unblended_cost_amount | rank |
+-----------+-------------+---------------------------+-----------------------+------+
| Name      |             | 2022-02-15T05:30:00+05:30 | $231.97               | 1    |
| Name      |             | 2022-02-16T05:30:00+05:30 | $122.25               | 2    |
| Name      |             | 2022-02-14T05:30:00+05:30 | $107.32               | 3    |
| Name      |             | 2022-10-17T05:30:00+05:30 | $88.94                | 4    |
| Name      |             | 2022-02-09T05:30:00+05:30 | $73.13                | 5    |
| Name      |             | 2022-10-18T05:30:00+05:30 | $60.92                | 6    |
| Name      |             | 2022-06-28T05:30:00+05:30 | $55.92                | 7    |
| Name      |             | 2022-06-27T05:30:00+05:30 | $54.58                | 8    |
| Name      |             | 2022-06-24T05:30:00+05:30 | $54.52                | 9    |
| Name      |             | 2022-06-26T05:30:00+05:30 | $54.50                | 10   |
+-----------+-------------+---------------------------+-----------------------+------+

Time: 20611.9ms. Rows fetched: 5605. Hydrate calls: 5605.
```

My aggregator connection name is `aws_all` where I have configured two account.

```
connection "aws" {
  plugin = "aws"
  regions = ["*"]
  profile = "aws_nagraj"
  max_error_retry_attempts = 10
  min_error_retry_delay = 50
}

connection "aws_aab" {
  plugin = "aws"
  regions = ["*"]
  profile = "aws-aab"
}

 connection "aws_all" {
   plugin      = "aws"
   type        = "aggregator"
   connections = ["aws", "aws_aab"]
 }
```

Not sure, if I doing anything wrong here?

Reply:
working for me now